ConnectivityDemos/ConnectivityDemosTest/pom.xml

```diff
@@ -14,8 +14,8 @@
   <packaging>iar</packaging>
 
   <properties>
-    <wss4j.version>2.2.3</wss4j.version>
-    <jersey.version>2.29.1</jersey.version>
+    <wss4j.version>2.2.5</wss4j.version>
```

Comment:
@ivy-ohe @alexsuter 
I think these versions refer to libraries we have in the ivy-core .... so they should be since with whats in the actual product. more often that not: this is not the latest version available.

The reason why these libs exists are either to extend the classpath with compatible addons to the ivy-core or to make classes accessible that are hidden in the axon.ivy container ... but actually accessible at runtime.

Reply:
huuu boy so I struggled the whole time for nothing :frowning: 

this would mean that the versionCheck is now more or less useless on those repositories... of course I can revert the version changes and then reset the quality gate but how useful is that then?

Reply:
Versions in core/thirdparty-libs:
wss4j = 2.2.4
jersey = 2.29.1
assertj-core = 3.12.2
junit = 5.6.0
assertj = 3.15.0
awaitility = 4.0.2

Reply:
I think the main goal was that we always use the latest project-build-plugin which is a plugin dependency. These are "normal maven dependencies". You can narrow the focus on plugin dependencies, for now.

Reply:
I turned the version check for "normal" dependencies off and only check plugins now. I set the versions to the same ones I found in the core